requirements.txt

```diff
@@ -1,25 +1,54 @@
+aiofiles==0.5.0
```

Comment:
Could you check if all these requirements needed? I see some requirements such as graphene or graphql won't be used.

Reply:
Since most of the packages were installed via server generation, I'm not sure what is required or not. I used the package pip-check-reqs to check what packages are currently unused and removed the ones that were not previously there. I also went on PyPi to briefly look at their descriptions before removing them.

Reply:
Looks good. Much cleaner!